### ISSUE TITLE
ERROR no prototype names matched 'kubeflow/core'

### ISSUE THREAD

**Issue by gaoning777** [kubeflow/kubeflow#1263]
I'm trying to install kubeflow by following the instructions https://www.kubeflow.org/docs/started/getting-started-gke/. The error comes when "ks generate kubeflow-core kubeflow-core --jupyterHubAuthenticator iap" is run.

I checked the ks prototypes by running "ks prototype list" but did not find the kubeflow-core. I made sure that I ran "ks pkg install kubeflow/core".

Any fixes?

**Comment by gaoning777** [comment]
I went back to previous commits of the kubeflow project and copied the all.jsonnet and all.libsonnet files to my local ks environment. Good thing is that kubeflow-core is found by ks but error is "ERROR find objects: RUNTIME ERROR: Field does not exist: all" when "ks apply default -c kubeflow-core" is run.

**Comment by jlewi** [comment]
deploy.sh will pull the latest source code from master which is not consistent with the 0.2.2 deploy script.

To fix this you need to set the environment variable `KUBEFLOW_VERSION`

```
export KUBEFLOW_VERSION=0.2.2
curl https://raw.githubusercontent.com/kubeflow/kubeflow/v${KUBEFLOW_VERSION}/scripts/deploy.sh | bash
```

There is an issue to fix this #1262 1239

**Comment by gaoning777** [comment]
Thanks a lot. that is exactly the problem.
